src/Lower.h

```diff
@@ -18,9 +20,9 @@ class IRMutator;
 /** Given a halide function with a schedule, create a statement that
  * evaluates it. Automatically pulls in all the functions f depends
  * on. Some stages of lowering may be target-specific. */
-EXPORT Stmt lower(const std::vector<Function> &output_funcs,
-                                  const std::string &pipeline_name, const Target &t,
-                  const std::vector<IRMutator *> &custom_passes = std::vector<IRMutator *>());
+ EXPORT Stmt lower(const std::vector<Function> &output_funcs, const std::string &pipeline_name,
```

Comment:
It's weird that this returns a Stmt and takes an inserter for submodules. Shouldn't it just return a Module, or mutate a Module argument?